### ISSUE TITLE
NVDA not reading the checkbox state correctly. Always reads as not checked.

### ISSUE THREAD

**Issue by Hardhik** [angular-ui/ui-grid#7095]
- [x] This is a bug report, not a question on how to use the grid.  
- [x] You have searched the open issues to see if this bug has been filed before.  We don't want duplicate issues.
- [x] You have reviewed the tutorials and documentation to ensure you are using the grid correctly. [ui-grid.info](http://ui-grid.info/docs/#/tutorial)
- [x] You are using the latest version of the grid. Older versions are not patched or supported.
- [x] You have provided steps to recreate your bug.  A plunkr is even better.

http://ui-grid.info/docs/#!/tutorial/Tutorial:%20121%20Grid%20Menu

Here's the gif from the tutorials site. As you can see the screen reader always reads it as not checked irrespective of the checkbox being checked or unchecked.

![uigrid](https://user-images.githubusercontent.com/2673006/87134713-ae294c80-c2b6-11ea-8c52-6ba9c9db2f8d.gif)

Expected behavior: 
The uigrid should have elements and attributes to correctly read out the status of the checkbox for screen readers.

**Comment by Hardhik** [comment]
Can someone please triage this. It's been a while since this has been filed.

**Comment by Hardhik** [comment]
Pinging on this. Is there any ETA on this ?

**Comment by Scrum** [comment]
@Hardhik Hi, I don’t think this is the nearest time, you can feel free about PR

**Comment by ashutosh100** [comment]
This bug is fixed in the latest update of angular ui grid
